### ISSUE TITLE
Can't use Sherlock with custom controls

### ISSUE THREAD

**Issue by lexer** [JakeWharton/ActionBarSherlock#561]
Sherlock cause inflate error when layout holds some custom control not included to default android control set.

For example GridLayout from support package.

java.lang.RuntimeException: Unable to start activity ComponentInfo{......activities.SettingsActivity}: android.view.InflateException: Binary XML file line #42: Error inflating class android.support.v7.widget.GridLayout
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:1654)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:1670)
        at android.app.ActivityThread.access$1500(ActivityThread.java:117)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:931)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:130)
        at android.app.ActivityThread.main(ActivityThread.java:3695)
        at java.lang.reflect.Method.invokeNative(Native Method)
        at java.lang.reflect.Method.invoke(Method.java:507)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:842)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:600)
        at dalvik.system.NativeStart.main(Native Method)
        Caused by: android.view.InflateException: Binary XML file line #42: Error inflating class android.support.v7.widget.GridLayout
        at android.view.LayoutInflater.createView(LayoutInflater.java:518)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:570)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:623)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:626)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:408)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:320)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:276)
        at com.actionbarsherlock.internal.ActionBarSherlockCompat.setContentView(ActionBarSherlockCompat.java:857)
        at com.actionbarsherlock.app.SherlockFragmentActivity.setContentView(SherlockFragmentActivity.java:251)
        at com.........activities.SettingsActivity.onCreate(SettingsActivity.java:47)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1047)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:1618)
        ... 11 more

**Comment by JakeWharton** [comment]
I don't see how ABS is affecting this. GridLayout works just fine for me. Are you sure you included it as a library project properly?

**Comment by JakeWharton** [comment]
Included the v7 support library as a library project, that is.

**Comment by lexer** [comment]
Im using v7 gridlayout support as maven dependency. So I think that Sherlock needs to have it in own dependencies to work correct, but I cannot add it.

**Comment by JakeWharton** [comment]
Is the v7 library in central or did you deploy it locally? There shouldn't be any overlap.

**Comment by lexer** [comment]
No. I have installed it using maven android sdk deployer: https://github.com/mosabua/maven-android-sdk-deployer/

I think the problem is not related directly to this lib.

I've got my own custom control that is included into primary android project and I also get inflate error on it.

Im not big expert of maven, but it seems that during inflate process it only see direct dependencies of sherlock library, but my own control and gridlayout are not there.
